Hardcore.lua

```diff
@@ -191,12 +210,12 @@ local function SlashHandler(msg, editbox)
 	elseif cmd == "debug" then
 		debug = not debug
 		Hardcore:Print("Debugging set to " .. tostring(debug))
-	elseif cmd == "notify" then
-		Hardcore_Settings.notify = not Hardcore_Settings.notify
-		if true == Hardcore_Settings.notify then
-			Hardcore:Print("Notification enabled")
+	elseif cmd == "alerts" then
```

Comment:
Should add "alerts"  to the /hc help command below

Reply:
I'll add to this asap